src/Sylius/Bundle/AdminBundle/Resources/config/grids/shipment.yml

```
@@ -6,9 +6,12 @@ sylius_grid:
                 options:
                     class: "%sylius.model.shipment.class%"
             fields:
```

I think we are missing this configuration:
```suggestion
             sorting:
                 createdAt: desc
             fields:
```

Ref. https://github.com/Sylius/Sylius/blob/58f63c7b95f55a951c7e41995494e20f537eca7d/src/Sylius/Bundle/AdminBundle/Resources/config/grids/admin_user.yml#L8-L9